### DISCUSSION TITLE
Support for Localization

### DISCUSSION THREAD

**Discussion by uDEV2019** [slowscript/warpinator-windows#2] [Ideas]
It would be good, when localization is possible before the first release.
Translating should be immedate possible, because the Strings should mostly be the same as the Android-Version.

**Comment by slowscript** [comment]
I have added support for localization today. It is slightly more work than on Android though, since you need Visual Studio and the controls don't resize themselves to fit the new text. Instructions on how to do it exactly are in the readme.

You've also mentioned that the strings should be the same (which they mostly are). Did you mean by that that I should copy them here? It is possible but the translations will be incomplete and will require cooperation from the translators anyway. Also I'm not entirely sure how to properly credit them for their contribution or even if I have the right to do this at all.
